From cfa40cc1c2df6c64fb53bb4a2a56eb09e3e1309c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 4 Jun 2021 18:21:58 -0700
Subject: [PATCH] core-net: Remove unused variable

variable 'n' is being set but it is not used anywhere, latest clang is
able to detect this and flags it

Fixes

lib/core-net/route.c:41:6: error: variable 'n' set but not used [-Werror,-Wunused-but-set-variable]
|         int n = 0;
|             ^

Upstream-Status: Submitted [https://github.com/warmcat/libwebsockets/pull/2320]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 lib/core-net/route.c | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/lib/core-net/route.c b/lib/core-net/route.c
index efe08ba1..6be93d62 100644
--- a/lib/core-net/route.c
+++ b/lib/core-net/route.c
@@ -38,23 +38,22 @@ void
 _lws_routing_entry_dump(lws_route_t *rou)
 {
 	char sa[48], fin[192], *end = &fin[sizeof(fin)];
-	int n = 0;
 
 	if (rou->dest.sa4.sin_family) {
 		lws_sa46_write_numeric_address(&rou->dest, sa, sizeof(sa));
-		n += lws_snprintf(fin, lws_ptr_diff_size_t(end, fin),
+		lws_snprintf(fin, lws_ptr_diff_size_t(end, fin),
 				  "dst: %s/%d, ", sa, rou->dest_len);
 	}
 
 	if (rou->src.sa4.sin_family) {
 		lws_sa46_write_numeric_address(&rou->src, sa, sizeof(sa));
-		n += lws_snprintf(fin, lws_ptr_diff_size_t(end, fin),
+		lws_snprintf(fin, lws_ptr_diff_size_t(end, fin),
 				  "src: %s/%d, ", sa, rou->src_len);
 	}
 
 	if (rou->gateway.sa4.sin_family) {
 		lws_sa46_write_numeric_address(&rou->gateway, sa, sizeof(sa));
-		n += lws_snprintf(fin, lws_ptr_diff_size_t(end, fin),
+		lws_snprintf(fin, lws_ptr_diff_size_t(end, fin),
 				  "gw: %s, ", sa);
 	}
 
-- 
2.31.1

